src/stores/wpcom-api.ts

```diff
@@ -43,12 +43,35 @@ const snapshotUsageSchema = z
 		siteCreationBlocked: data.site_creation_blocked,
 	} ) );
 
+const blueprintSchema = z.object( {
+	slug: z.string(),
+	title: z.string(),
+	excerpt: z.string(),
+	image: z.string(),
+	playground_url: z.string(),
+	blueprint: z.record( z.unknown() ),
+} );
+
+const blueprintsResponseSchema = z.object( {
+	blueprints: z.array( blueprintSchema ),
+	total: z.number(),
+} );
+
 let wpcomClient: WPCOM | undefined;
+let publicWpcomClient: WPCOM | undefined;
 
 export const setWpcomClient = ( client: WPCOM | undefined ) => {
 	wpcomClient = client;
 };
 
+// Get or create a public WPCOM client for unauthenticated requests
+function getPublicWpcomClient(): WPCOM {
+	if ( ! publicWpcomClient ) {
+		publicWpcomClient = new WPCOM();
+	}
+	return publicWpcomClient;
+}
+
```

Comment:
Why do we need the getPublicWpcomClient function?
Let's initialize the variable directly:

```suggestion
const publicWpcomClient = new WPCOM();

export const setWpcomClient = ( client: WPCOM | undefined ) => {
	wpcomClient = client;
};

```

And we can move it closer to its first usage.